tasks/prime.py

```diff
@@ -362,6 +369,48 @@ def create_mto_shipment(self, overrides=None):
             self.add_stored(MTO_SHIPMENT, mto_shipment)
             return mto_shipment
 
+    @tag(MTO_SHIPMENT, "createMTOShipment", "expectedFailure")
+    @task
+    def create_mto_shipment_with_duplicate_agents(self, overrides=None):
+        # If moveTaskOrderID was provided, get that specific one. Else get any stored one.
+        object_id = overrides.get("moveTaskOrderID") if overrides else None
+
+        move_task_order = self.get_stored(MOVE_TASK_ORDER, object_id)
+        if not move_task_order:
+            logger.debug("createMTOShipment: ⚠️ No move_task_order found")
```

Comment:
For consistency I think only errors are using the caution symbol.  It may make things confusing if debug messages also start using it.

Reply:
Right now `⚠️` is being used on both error logging and when we gracefully terminate the load test within the `prime.py` file. This is a case where we are gracefully terminating.

Are you suggesting that we change this throughout the file? Since we shouldn't expect data issues, I think they should be flagged when they occur. Having some sort of icon helps make this more transparent. While I think ⚠️ is an appropriate fit, I am not inherently opposed to having a different message and only using ⚠️ for the load testing error messages during a request.

Reply:
You're right I see there are some other debug statements that are using that icon now.  For me it's been helpful to only locate the errors in the logs.  I think most of these debug cases are early in the locust tests before there is any data to work on depending on the sequence of the tasks that are called and the users that are spawned.  I think the message is helpful but might detract from the real error responses.